src/internal/ans.ts

```diff
@@ -465,7 +470,7 @@ export async function getAccountSubdomains(
     },
   });
 
-  return data.current_aptos_names;
+  return data.current_aptos_names.map(sanitizeANSName);
```

Comment:
do we need to validate `current_aptos_names` is not empty?

Reply:
Nope! The indexer always gives back an array, empty or not. `map` here will be a no-op if we get an empty array back. 

<img width="268" alt="image" src="https://github.com/aptos-labs/aptos-ts-sdk/assets/694324/dddc1b4c-35e2-4c8a-b6bd-7dc0f8dc537e">